Minifigures.UI/src/app/app.component.ts

```diff
@@ -10,10 +10,23 @@ import { BoardGameService } from './services/boardgame.service';
 export class AppComponent {
   title = 'Minifigures.UI';
   boardGames: BoardGame[] = [];
+  boardgameToEdit?: BoardGame;
 
   constructor(private boardGameService: BoardGameService) {}
 
   ngOnInit() : void{
     this.boardGameService.getBoardGames().subscribe((result: BoardGame[]) => (this.boardGames = result));
   }
+
+  updateBoardGameList(boardGames: BoardGame[]) {
+    this.boardGames = boardGames;
+  }
+
+  initNewBoardGame() {
```

Comment:
🌱 Lets start using return types in frontend! Even if its void, then its worth to add it to all of the functions